src/less/textbox/textbox.less

```diff
@@ -80,22 +80,22 @@ textarea.textbox__control {
     &:focus {
         .border-color-token(textbox-focus-border-color, color-stroke-default);
         .background-color-token(textbox-focus-background-color, color-background-primary);
-        .color-token(textbox-focus-foreground-color, color-foreground-secondary);
+        .color-token(textbox-focus-foreground-color, color-foreground-on-secondary);
```

Comment:
Why can't this be the default? Always have the same `color` irrespective of whether focused or not?

Reply:
We can probably. 
But its mostly for tokens in case we need a fine grain tuning of the colors